docs/sphinx/requirements.txt

```diff
@@ -116,7 +116,7 @@ requests==2.28.2
     # via
     #   pygithub
     #   sphinx
-rocm-docs-core[api_reference]>=0.30.3
+rocm-docs-core[api_reference]>=0.31.0

```

Comment:
Should this also be 30.3? Or is there a reason for bumping down to 31.0?

Reply:
@charis-poag-amd Can merge this in here? I would like to avoid uploading via Gerrit?